enterprise/kerberos-login-module/README.md

```diff
@@ -0,0 +1,224 @@
+# A Kerberos login module for Hazelcast.
+
+Requires a Hazelcast Enterprise licence. Download a trial key from
+
+http://hazelcast.com/hazelcast-enterprise-trial/
+
+## Introduction
+
+Demonstrates how to create a Kerberos Login Module for Hazelcast.
+
+The project builds and produces the `kerberos-login-module.jar` artifact. It relies on the [Java Kerberos Utilities library](https://github.com/jcmturner/java-kerberos-utils) which is available under the Apache License 2.0. The library is included in the project's `lib` directory, and embedded in the produced artifact.
+
+The module relies on Java GSS and therefore requires Oracle/OpenJDK JVM.
```

Comment:
The JGSS is available in IBM JDKs too, the problem is in using private `com.sun.security` and `sun.security` packages. So I would choose another wording here. E.g.

> The module relies on internal Java GSS implementation classes in `com.sun.security` and `sun.security` packages and therefore requires Oracle/OpenJDK JVM.

Reply:
Fixing